src/styles/index.css

```diff
@@ -0,0 +1,204 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.calendar {
+  margin-left: 77px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: 82px;
+  width: 870px;
+}
+
+.calendar__day {
+  display: flex;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+  height: 100px;
+  width: 100px;
+  border: 1px solid #000;
+  background-color: #eee;
+  transition-duration: 0.5s;
+  align-items: center;
+  justify-content: center;
+}
+
+.calendar__day:hover {
+  background-color: #ffbfcb;
+  transform: translate(0, -20px);
+}
+
+.calendar__day:nth-child(1)::before {
+  content: "1";
+}
+
+.calendar__day:nth-child(2)::before {
+  content: "2";
+}
+
+.calendar__day:nth-child(3)::before {
+  content: "3";
+}
+
+.calendar__day:nth-child(4)::before {
+  content: "4";
+}
+
+.calendar__day:nth-child(5)::before {
+  content: "5";
+}
+
+.calendar__day:nth-child(6)::before {
+  content: "6";
+}
+
+.calendar__day:nth-child(7)::before {
+  content: "7";
+}
+
+.calendar__day:nth-child(8)::before {
+  content: "8";
+}
+
+.calendar__day:nth-child(9)::before {
+  content: "9";
+}
+
+.calendar__day:nth-child(10)::before {
+  content: "10";
+}
+
+.calendar__day:nth-child(11)::before {
+  content: "11";
+}
+
+.calendar__day:nth-child(12)::before {
+  content: "12";
+}
+
+.calendar__day:nth-child(13)::before {
+  content: "13";
+}
+
+.calendar__day:nth-child(14)::before {
+  content: "14";
+}
+
+.calendar__day:nth-child(15)::before {
+  content: "15";
+}
+
+.calendar__day:nth-child(16)::before {
+  content: "16";
+}
+
+.calendar__day:nth-child(17)::before {
+  content: "17";
+}
+
+.calendar__day:nth-child(18)::before {
+  content: "18";
+}
+
+.calendar__day:nth-child(19)::before {
+  content: "19";
+}
+
+.calendar__day:nth-child(20)::before {
+  content: "20";
+}
+
+.calendar__day:nth-child(21)::before {
+  content: "21";
+}
+
+.calendar__day:nth-child(22)::before {
+  content: "22";
+}
+
+.calendar__day:nth-child(23)::before {
+  content: "23";
+}
+
+.calendar__day:nth-child(24)::before {
+  content: "24";
+}
+
+.calendar__day:nth-child(25)::before {
+  content: "25";
+}
+
+.calendar__day:nth-child(26)::before {
+  content: "26";
+}
+
+.calendar__day:nth-child(27)::before {
+  content: "27";
+}
+
+.calendar__day:nth-child(28)::before {
+  content: "28";
+}
+
+.calendar__day:nth-child(29)::before {
+  content: "29";
+}
+
+.calendar__day:nth-child(30)::before {
+  content: "30";
+}
+
+.calendar__day:nth-child(31)::before {
+  content: "31";
```

Comment:
Instead of repeating the same selector for each day, consider using a loop in a preprocessor like SASS or LESS to generate these selectors. This will make your code more maintainable and reduce redundancy.